packages/components/src/components/Table/Table.js

```diff
@@ -161,7 +166,8 @@ Table.defaultProps = {
   batchActionButtons: [],
   toolbarButtons: [],
   loading: false,
-  isSortable: false,
+  useZebraStyles: true,
```

Comment:
variables declared here are ones that might be passed in as props. is `useZebraStyles` going to be passed as prop?

Reply:
Changed this now :)